docs/book/README.md

```diff
@@ -29,12 +29,11 @@ This document will cover both the in-tree and out-of-tree vSphere integrations f
 
 ## Tutorials
 
-### vSphere 6.7U3 specific tutorials
+### vSphere 6.7U3 tutorials
 
 * [Deploying a new K8s cluster with CPI and CSI on vSphere 6.7U3 with kubeadm](./tutorials/kubernetes-on-vsphere-with-kubeadm.md)
 * [Deploying CPI and CSI with Zones Topology](./tutorials/deploying_cpi_and_csi_with_multi_dc_vc_aka_zones.md)
 
 ### Earlier tutorials
 
-* [Deploying the CSI and CCM on pre-6.7U3](./tutorials/deploying_cloud_provider_vsphere_with_rbac.md)
```

Comment:
What made this tutorial pre 6.7U3?

Reply:
Those were the pre-combined CPI/CSI instructions where you could have used the CPI and the pre-CNS-based CSI driver to deploy. Since we are not going to support before 6.7u3 for CPI/CSI, we might as well remove all the old supporting documentation.

Reply:
gotcha